docs/persistent-subscriptions/README.md

```diff
@@ -0,0 +1,105 @@
+# Persistent subscriptions
+
+A common operation is to subscribe to a stream and receive notifications for changes. As new events arrive, you continue following them. 
+
+You can only subscribe to one stream or the $all stream. You can use server-side projections for linking events to new aggregated streams. System projections create pre-defined streams that aggregate events by type or by category and are available out-of-the box. Learn more about system and user-defined projections (here)[../projections/]
```

Comment:
We should update the docs around persistent subscription to `$all` - they're not available in the v20.10.